### PR TITLE
added function to get demography debugger from a demographic model

### PR DESCRIPTION
**Pull request by andrewkern**

### CODE REVIEW

File: stdpopsim/models.py [hunk 1]
```diff
@@ -276,6 +276,17 @@ def get_samples(self, *args):
                                  f" {pop_index}")
         return samples
 
+    def get_demography_debugger(self):
+        """
+        Returns an msprime DemographyDebugger initialize
```
Comment (grahamgower) [inline stdpopsim/models.py]: s/initialize/initialized/

File: tests/test_models.py [hunk 2]
```diff
@@ -453,3 +453,10 @@ def test_population_config_order_equal(self):
                 config.metadata["name"] for config in model.population_configurations]
             for p, c in zip(pop_names, config_names):
                 self.assertEqual(p, c)
+
+    # Test that we are indeed getting a valid DDB back
+    # admittedly a pretty bad test...
+    def test_demography_debugger(self):
+        for model in stdpopsim.all_demographic_models():
+            ddb = model.get_demography_debugger()
+            self.assertEqual(type(ddb), msprime.DemographyDebugger)
```
Comment (grahamgower) [inline tests/test_models.py]: You could instead use `self.assertIsInstance()`.

Comment (andrewkern) [reply tests/test_models.py]: sure no prob

File: stdpopsim/models.py [hunk 3]
```diff
@@ -276,6 +273,17 @@ def get_samples(self, *args):
                                  f" {pop_index}")
         return samples
 
+    def get_demography_debugger(self):
+        """
+        Returns an msprime DemographyDebugger initialized
```
Comment (jeromekelleher) [inline stdpopsim/models.py]: We can link to the docs in msprime using sphinx. Something like

```
Returns an :class:`msprime.DemographyDebugger` instance initialized with the parameters 
for this model. Please see the msprime documentation for details on how to use a 
DemographyDebugger.

:return: A DemographyDebugger instance for this DemographicModel.
:rtype: msprime.DemographyDebugger
```

File: tests/test_models.py [hunk 4]
```diff
@@ -453,3 +453,10 @@ def test_population_config_order_equal(self):
                 config.metadata["name"] for config in model.population_configurations]
             for p, c in zip(pop_names, config_names):
                 self.assertEqual(p, c)
+
+    # Test that we are indeed getting a valid DDB back
+    # admittedly a pretty bad test...
```
Comment (jeromekelleher) [inline tests/test_models.py]: Could do something like
```python
other_ddb = msprime.DemographyDebugger(
￼            population_configurations=model.population_configurations,
￼            migration_matrix=model.migration_matrix,
￼            demographic_events=model.demographic_events)
f1 = io.StringIO()
f2 = io.StringIO()
ddb.print_history(f1)
other_ddb.print_history(f2)
self.assertEqual(f1.getvalue(), f2.getvalue())
```

File: stdpopsim/models.py (hunk 3 above)
Comment (andrewkern) [reply stdpopsim/models.py]: yes- i need to get in this habit! change made

File: tests/test_models.py (hunk 4 above)
Comment (andrewkern) [reply tests/test_models.py]: implemented this as a second test

**Comment by andrewkern** [comment]
okay should be ready to merge @jeromekelleher

**Comment by jeromekelleher** [comment]
Nice, thanks @andrewkern.

**Pull request merged**